src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -723,28 +703,34 @@ describe("Decimal", () => {
       describe("precision", () => {
         it("fires a console error when precision is changed once component is loaded.", () => {
           jest.spyOn(global.console, "error").mockImplementation(() => {});
-          render({ defaultValue: "12345", precision: 2 });
+          render({ defaultValue: "12345", precision: 2, ...itProps });
           setProps({ precision: 1 });
           expect(console.error).toHaveBeenCalledWith(
             "Decimal `precision` prop has changed value. Changing the Decimal `precision` prop has no effect."
           );
         });
 
         it("supports a precision of 0", () => {
-          render({ defaultValue: "12345", precision: 0 });
+          render({ defaultValue: "12345", precision: 0, ...itProps });
           expect(value()).toBe("12.345");
           expect(hiddenValue()).toBe("12345");
         });
 
+        it("has a default precision of 2 (crop)", () => {
```

Comment:
Test will need a different description as the value isn't cropped and the value shouldn't be cropped either.